client/state/selectors/get-request.js

```diff
@@ -0,0 +1,22 @@
+/** @format */
+/**
+ * Internal dependencies
+ */
+import { getRequestKey } from 'state/data-layer/wpcom-http/utils';
+
+/**
+ * Returns meta information about data requests going through the data layer
+ *
+ * @param {Object} state Redux state
+ * @param {Object} action data request action
+ * @returns {*} metadata about request
+ */
+export default ( state, action ) => {
+	const data = state.dataRequests[ getRequestKey( action ) ] || {};
+
+	return {
+		...data,
+		hasLoaded: data.lastUpdated > -Infinity,
```

Comment:
this is absurdly nitty, but `-Infinity` looks silly considering we don't have a notion of negative time...in this universe at least.  

would `hasLoaded: isNumber( data.lastUpdated )` work just as well?

Reply:
> considering we don't have a notion of negative time...in this universe at least.

Tell that to the people who designed and standardized the timestamp.

```js
const t = (new Date('1903-06-14 00:00:00')).getTime()

t === -2100196800000
```